07-routing-gossip.md

```diff
@@ -445,10 +445,12 @@ or `node_id_2` otherwise.
 ### Requirements
 
 The origin node:
-  - MUST NOT send a created `channel_update` before `funding_locked` has been received.
+  - MUST NOT send a created `channel_update` before `channel_ready` has been received.
   - MAY create a `channel_update` to communicate the channel parameters to the
```

Comment:
I think once the `announcement_signatures` has been exchanged, each side should send the `channel_update` with the real SCID to each other since prior to this point, they might have been using alias SCIDs in the `channel_update`. Unless this is already implied since both sides will broadcast (to each other, the network) the "real" `channel_update`?